### ISSUE TITLE
Getting errors when trying to apply image transforms

### ISSUE THREAD

**Issue by keithmancuso** [markhuot/craftql#78]
I have this setup in a query
```
 featuredImage {
        thumbnail: url(transform: facebook)
        url
        title
        mimeType
      }
```
getting errors on the facebook line? I know it recognized the transform handle tho because it type-ahead-ed?  

```
  "errors": [
    {
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 10,
          "column": 9
        }
      ],
      "path": [
        "entry",
        "featuredImage",
        0,
        "thumbnail"
      ]
    }
  ],

```
Any ideas whats up with that?

**Comment by markhuot** [comment]
Just to double check, can you see if the `crop:` argument works? I don’t use named transforms so there could definitely be a bug here.

**Comment by markhuot** [comment]
`dev-master` should fix this issue. The upstream GraphQL library changed how Enums worked which caused the handle to be used incorrectly.

**Comment by markhuot** [comment]
This was released with 1.0.0
